CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Changelog
 
+## 0.2.0 (2023-02-24)
+
+- Added Miden Assembly instructions to the Playground
+- Added ability to prove a program to the Playground
+- Refactored the frontend using TailwindUI
```

Comment:
nit: I would change this to "Refactored the front-end to use the Tailwind css framework ", since tailwind is the design framework and TailwindUI is a service that provides pre-designed components.